docs/migration_guide.md

```diff
@@ -0,0 +1,127 @@
+# Migration guide from YAML payloads to JSON (bfb) protocol
+
+## Background
+Communication protocol between SUA and backend was extended to support vehicle orchestration for large quantities of devices. Key differences are:
+* Transition from YAML to JSON
+* Fine-grained API (command-based approach instead of single-shot self-update)
+* Extension of statuses (for detailed reflection of success, progress or failure)
+Some fiels are not relevant anymore meanwhile new were introduced. Below is a brief comparison between YAML and JSON messages.
+
+### Transition guide for current state message
+From the current state message in YAML payload there is only 1 field 'bundleVersion' which is relevant and corresponding field in JSON is 'version' in 'softwareNodes' for the device image (actual name could differ depending on distro configuration):
```

Comment:
better: "only one field",
"...is relevant. The corresponding field...", 
"image. (The actual name..."

Reply:
Fixed